tracer/transport.go

```diff
@@ -22,6 +22,12 @@ type Transport interface {
 	SetHeader(key, value string)
 }
 
+// NewHTTPTransport returns a new Transport implementation using the given
+// delivery url.
+func NewHTTPTransport(url string) Transport {
```

Comment:
May be better to have this take a `hostname, port` instead, to expose it a little less details about the URL structure. Similar to how the ruby lib uses a `trace.configure(hostname: 'foo', port: '7777')`.